### PR TITLE
N21 1774 create synced course

### PR DESCRIPTION
**Pull request by IgorCapCoder**
# Description
add Feature flag FEATURE_SCHULCONNEX_COURSE_SYNC_ENABLED for nbc (only dev atm)

## Links to Tickets or other pull requests
- https://ticketsystem.dbildungscloud.de/browse/N21-1774
- https://github.com/hpi-schul-cloud/schulcloud-client/pull/3426
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/4830

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -984,6 +984,10 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  FEATURE_SCHULCONNEX_COURSE_SYNC_ENABLED:
+    server: true
+    client: true
```
Comment (arnegns) [inline ansible/group_vars/all/config.yml:989]: you really need this in client?

**Review by arnegns — APPROVED** [review]

File: ansible/group_vars/all/config.yml [hunk 2]
```diff
@@ -984,6 +984,10 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  FEATURE_SCHULCONNEX_COURSE_SYNC_ENABLED:
```
Comment (arnegns) [inline ansible/group_vars/all/config.yml:987]: Is this new env documented?

**Review by arnegns — APPROVED** [review]

File: ansible/group_vars/all/config.yml (hunk 1 above)
Comment (IgorCapCoder) [reply ansible/group_vars/all/config.yml:989]: yes, the legacy client checks for this very feature flag

File: ansible/group_vars/all/config.yml (hunk 2 above)
Comment (IgorCapCoder) [reply ansible/group_vars/all/config.yml:987]: yes, it is :)

**Pull request merged**